pkg/registry/resourcequota/strategy.go

```diff
@@ -106,9 +106,8 @@ func MatchResourceQuota(label labels.Selector, field fields.Selector) generic.Ma
 }
 
 // ResourceQuotaToSelectableFields returns a label set that represents the object
-// TODO: fields are not labels, and the validation rules for them do not apply.
 func ResourceQuotaToSelectableFields(resourcequota *api.ResourceQuota) labels.Set {
 	return labels.Set{
-		"name": resourcequota.Name,
+		"metadata.name": resourcequota.Name,
```

Comment:
As per my comment, I dont think we need to support name (external clients could not have used it anyway). Will discuss it with @bgrant0607 tomorrow. Not blocking this PR on that.